buy_together_app/templates/buy_together_app/main.html

```diff
@@ -1,62 +1,23 @@
+{% load static %}
 <!DOCTYPE html>
     <html lang="en">
     <head>
-        <title>buyTogether</title>
-        <style type="text/css">
-            body{
-                background-color: antiquewhite;
-            }
-            table {
-                font-family: arial, sans-serif;
-                border-collapse: collapse;
-                width: 100%;
-              }
-              
-              td, th {
-                border: 1px solid #dddddd;
-                text-align: left;
-                padding: 8px;
-              }
-              
-              tr:nth-child(even) {
-                background-color: #dddddd;
-              }
-              .projectName{
-                  text-align: center;
-                  font-size: 100px;
-                  color: purple;
-              }
-              .bigFont{
-                  font-size: 30px;
-                  margin-top: 80px;
-                  margin-left: 20px;
-                  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
-              }
-              .red{
-                  color: red;
-              }
-            .blue{
-                color: blue;
-            }
-
-        </style>
+        <title>{% block title %} buyTogether {% endblock %}</title>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+        <link rel="stylesheet" href="{% static 'css/website_main_page.css' %}">
     </head>
 
     <body>
-        <div class="projectName red">
-            buyTogether
+        {% include './parts/nav.html' %}
```

Comment:
I'm getting an error here: `TemplateDoesNotExist at /`
The given path is incorrect.

Reply:
For me it does work

Reply:
That doesn't make sense. there is no `parts` directory. Please check your path and files again.

Reply:
that is because parts was in gitignore,fixed!